### PR TITLE
chore: repo maintenance

### PR DESCRIPTION
**Pull request by jinwoo**
- Add the "files" stanza to package.json for npm publishing
- Code coverage support
- CircleCI setup
- License checking. `bloomfilter` must be whitelisted.
- Package updates.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          filters: *release_tags
+      - node8:
+          filters: *release_tags
+      - node9:
+          filters: *release_tags
+      - publish_npm:
+          requires:
+            - node4
+            - node6
+            - node8
```
Comment (ofrobots) [inline .circleci/config.yml]: Don't we support node 8+?

File: package.json [hunk 2]
```diff
@@ -21,14 +29,27 @@
...
+      "build/test/*-test.js"
+    ]
+  },
+  "nyc": {
+    "exclude": [
+      "build/test"
+    ]
+  },
   "dependencies": {
     "bloomfilter": "0.0.16"
```
Comment (ofrobots) [inline package.json]: This doesn't use a semver range. Intentional?

File: .circleci/config.yml (hunk 1 above)
Comment (jinwoo) [reply .circleci/config.yml]: Copy-pasta failure :) Fixed.

File: package.json (hunk 2 above)
Comment (jinwoo) [reply package.json]: Not intentional. Fixed.

**Pull request merged**